src/gamestate/commands.cpp

```diff
@@ -5177,6 +5178,8 @@ void execute_command(sys::state& state, payload& c) {
 	case command_type::c_complete_constructions:
 		execute_c_complete_constructions(state, c.source);
 		break;
+	case command_type::c_instant_research:
+		execute_c_instant_research(state, c.source);
```

Comment:
needs a break I think so that the next time we add a command someone doesn't introduce a bug